doc/classes/EditorScenePostImport.xml

```diff
@@ -28,9 +28,9 @@
 		// This sample changes all node names.
 		// Called right after the scene is imported and gets the root node.
 		[Tool]
-		public class NodeRenamer : EditorScenePostImport
+		public partial class NodeRenamer : EditorScenePostImport
 		{
-		    public override Object PostImport(Object scene)
+		    public override Object _PostImport(Object scene)
```

Comment:
So this was renamed and I can see it in `godot/editor/project_converter_3_to_4.cpp` but it's still saying there's `no suitable method found to override` when I try to compile this. I've checked in the docs and the engine and this should be correct

Reply:
The signature I'm seeing in the engine is:

```csharp
public virtual Object _PostImport(Node scene);
```

Reply:
🤦
Fixed it now